code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -219,12 +219,15 @@
 	return
 
 /datum/species/ipc/spec_life(mob/living/carbon/human/H)
-	. = ..()
-	if(H.health <= UNCONSCIOUS && H.stat != DEAD) // So they die eventually instead of being stuck in crit limbo.
-		H.adjustFireLoss(6) // After bodypart_robotic resistance this is ~2/second
+	H.setOxyLoss(0)
+	H.losebreath = 0
+	if(H.health <= UNCONSCIOUS && H.stat != DEAD && !HAS_TRAIT(H, TRAIT_NOCRITDAMAGE)) // So they die eventually instead of being stuck in crit limbo.
```

Comment:
```suggestion
	if(H.stat<= UNCONSCIOUS && H.stat != DEAD && !HAS_TRAIT(H, TRAIT_NOCRITDAMAGE)) // So they die eventually instead of being stuck in crit limbo.
```
bruh

Reply:
actually this isn't right too, check bacons comment

Reply:
Will fix